### ISSUE TITLE
can this tool convert pytorch model to caffe model?

### ISSUE THREAD

**Issue by 1976277169** [marvis/pytorch-caffe#9]
the example is using caffe model in pytorch derectly, can this tool convert pytorch model to caffe model?
and how? can you give me a hand? thx a lot~

**Comment by fy-meng** [comment]
I don't think it can. However, if you still have this problem, you can first [export your pytorch model to onnx](https://pytorch.org/docs/stable/onnx.html), which is natively supported by pytorch, and then use some scripts like [this repo](https://github.com/MTlab/onnx2caffe) to convert onnx into prototxt and caffemodel. Most commonly used operations are supported, but you may still encounter some problems if some less common operations exist.
